internal/repositories/user.go

```diff
@@ -29,7 +29,7 @@ func (r *userRepo) Create(username, displayName, password, email string) error {
 	if err != nil {
 		return response.ErrInternalServer
 	}
-	_, err = r.db.Exec("INSERT INTO users (username, displayName ,password, email) VALUES ($1, $2, $3, $4)", username, displayName, hashed, email)
+	_, err = r.db.Exec("INSERT INTO users (username, display_name ,password, email) VALUES ($1, $2, $3, $4)", username, displayName, hashed, email)
```

Comment:
```suggestion
	_, err = r.db.Exec("INSERT INTO users (username, display_name ,password, email) VALUES ($1, $2, $3, $4)", username, display_name, hashed, email)
```
こっちはスネークケースにできていないかも？

Reply:
直すぜい

Reply:
いや、これは変数名だからスネークケースじゃなくていいね。Goはキャメルケースが原則なのでに。

Reply:
なるほどね！
変数名を明記するのか

Reply:
[463b239](https://github.com/KDG-Develop-Hub/social-api/pull/8/commits/463b23973fd8341bab25117b542bc632fbdeca18)|
できちゃあ